### PR TITLE
misc. minor documentation changes

### PR DESCRIPTION
**Pull request by xenophenes**

### CODE REVIEW

File: docs/operator-docs.asciidoc [hunk 1]
```diff
@@ -806,28 +804,16 @@ You can view the backup:
 pgo show backup mycluster
 ....
 
-You can view the backup along with a PVC listing:
-....
-pgo show backup mycluster --show-pvc=true
-....
+View the PVC folder and the backups contained therein:
```
[the remark this answers is not in the document]
Comment (xenophenes) [reply docs/operator-docs.asciidoc]: If you see the commit message related to this, --show-pvc is no longer valid as confirmed by myself and Pramodh. I changed the code beneath to have alternative commands that show the backup paths.

File: docs/operator-docs.asciidoc [hunk 2]
```diff
@@ -806,28 +804,16 @@ You can view the backup:
...
+pgo show pvc mycluster-backup-pvc
+pgo show pvc mycluster-backup-pvc --pvc-root=mycluster-backups
+...
 
-database pod mycluster is found
-
-├── mycluster-backups/2017-03-27-13-54-33
-├── mycluster-backups/2017-03-27-13-56-49
-└── mycluster-backups/2017-03-27-14-02-38
-....
```
[the remark this answers is not in the document]
Comment (xenophenes) [reply docs/operator-docs.asciidoc]: The command is no longer valid.

File: docs/operator-docs.asciidoc [hunk 3]
```diff
@@ -14,14 +14,14 @@ from source, you can download them from the following:
...
-script useful https://github.com/CrunchyData/postgres-operator/blob/master/examples/quickstart.sh
+For most users of the Operator who simply want to deploy
+it on a running Kubernetes cluster will find the Quickstart
+script useful: https://github.com/CrunchyData/postgres-operator/blob/master/examples/quickstart.sh
 
 Executing that script will deploy the Operator to your Kube
 cluster and also set up your local user environment with the
 *pgo* client in about 10 seconds.  More details on the Quickstart
-is at the end of this document <<Quickstart>>.
+script is located in the <<Quickstart>> section in this document.
```
[the remark this answers is not in the document]
Comment (xenophenes) [reply docs/operator-docs.asciidoc]: That seems to be a link format; it is "linked" in that manner in the current version of the document and it takes you down to the Quickstart section.

File: docs/operator-docs.asciidoc [hunk 4]
```diff
@@ -14,14 +14,14 @@ from source, you can download them from the following:
 Further details can be found in the link:design.asciidoc[PostgreSQL Operator Design] document on
 how the operator is built and how it operates.
 
-For most users of the Operator who simply want to deploy 
-it on a running Kube cluster will find the Quickstart
-script useful https://github.com/CrunchyData/postgres-operator/blob/master/examples/quickstart.sh
+For most users of the Operator who simply want to deploy
+it on a running Kubernetes cluster will find the Quickstart
+script useful: https://github.com/CrunchyData/postgres-operator/blob/master/examples/quickstart.sh
```
[the remark this answers is not in the document]
Comment (xenophenes) [reply docs/operator-docs.asciidoc]: It's automatically converted into a link, if you view the operator-docs document.

[the remark this answers is not in the document]
Comment (abrightwell) [reply docs/operator-docs.asciidoc]: That's not what I mean. What I mean is something less verbose. What I mean is, `link:<target>[text]`. See example in documentation above this instance.

File: docs/operator-docs.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (abrightwell) [reply docs/operator-docs.asciidoc]: Sounds good. Can you amend your commit message to include the actual commit ID that removed this feature?

[the remark this answers is not in the document]
Comment (xenophenes) [reply docs/operator-docs.asciidoc]: Yep, updated!

File: docs/operator-docs.asciidoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (xenophenes) [reply docs/operator-docs.asciidoc]: Updated!

**Comment by xenophenes** [comment]
Latest commit has the following updates:

- Added a step to source your .bashrc file, similar to what's in crunchy-containers
- Added a section documenting the various username/password combinations created by default, and where they're created from (in the event the user wants to change these values or know how to authenticate). Previously, no mention was made of the passwords configured or the default accounts.
- Changed wording minorly in the Quickstart section to make it more clear the quickstart script as-is will not work on a standard Kubernetes deployment; it will need to be modified before it will work.

**Pull request merged**